server/server.go

```diff
@@ -128,6 +131,10 @@ func NewServer(ctx *Context, stopper *util.Stopper) (*Server, error) {
 			return nil, err
 		}
 	}
+
+	//
+	s.sqlDB = sqlserver.NewDBServer(s.db)
```

Comment:
sql/sqlserver is definitely a stutter. These packages should be sql/server and sql/wire